lib/docs/src/tag.stories.tsx

```diff
@@ -1,24 +1,54 @@
 import { Meta } from "@storybook/react";
 import { Fragment } from "react";
-import { DivPx, Tag } from "../../core/src";
+import { Button, DivPx, Tag } from "../../core/src";
+import { Utils } from "./utils";
 
-export default {
-	title: "Draft/Tag",
+const colors: string[] = [
+	"red",
+	"yellow",
+	"green",
+	"blue",
+	"indigo",
+	"purple",
+	"pink",
+	"gray",
+];
+
+const meta: Meta = {
+	title: "Components/Tag",
 	component: Tag,
-} as Meta;
-
-export const Primary = (): JSX.Element => {
-	const colors: string[] = [
-		"red",
-		"yellow",
-		"green",
-		"blue",
-		"indigo",
-		"purple",
-		"pink",
-		"gray",
-	];
+	argTypes: {
+		children: Utils.arg("text"),
+		color: Utils.arg(colors),
+	},
+};
+
+Utils.page.component(meta, { sticky: true, shots: [] });
+
+export default meta;
+
+interface Props {
+	children?: string;
+	color?: string;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<Tag color={(Tag.colors as any)[props.color ? props.color : "gray"]}>
+			{props.children ?? "Default"}
+		</Tag>
+	);
+};
 
+export const Basic = (): JSX.Element => {
+	return <Tag color={Tag.colors.green}>Tag</Tag>;
+};
+
+Utils.desc(Basic)(`
+Tags are used to provide additional information about something. To begin, you need to provide a color from Tag.color and some information as string via children.
+`);
+
+export const Usage = (): JSX.Element => {
```

Comment:
I'm sorry but I think we can drop this section:
- The colors of Tags should be shown much easier, using the "Gallery" section (see my comment above regarding the "shots" option)
- The usage of Tags inside Buttons is not really recommended at the moment. I'm leaning to put it outside of other components, like next to Buttons, instead of inside them